COPYING.md

```diff
@@ -0,0 +1,232 @@
+# License Agreement/Contract(s)
+**Severability/salvatorius: If any word, part, portion, provision , of this is or should become illegal, invalid, null or void, rendering this unenforceable in any jurisdiction, that then shall not affect:** 
+
++ the validity or enforceability in that jurisdiction of any other provision of this Agreement; nor the validity or enforceability in any other types of jurisdictions on that or of any other provision of this Agreement.
+- Any other part or character of the license/contract.
+
+
+#### (C) Everyone is permitted to copy and distribute verbatim or modified copies of this license document, and changing it is allowed
+
+
+This software is licensed with multiple options, latest version of either: MIT license or (/and) Apache license .
+
+## In-Addition To:
++ Explicit Grant of Patent License or any relevent sublicense (/other) rights.
```

Comment:
Ditto.
Plus I don't understand his point, or rather I don't understand implications of this point. Isn't all together creating another license than either MIT or Apache/2.0?

Reply:
Nope, the implications of another license could already be considered to be there in the current file. This file serves as clarification and removes the doubts on implicit patent rights across jurisdictions that were raised in Issue #5, this copying file also clarifies that people can have either license or both. (To the extent that is permitted by relevant jurisdictions & Copyright)

Reply:
As permissive as can be and allows amendments with non-kerla code that don't violate the rules established in either (or both) licenses depending on the repo that's hosting it